x-pack/plugins/cases/public/methods/can_use_cases.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ApplicationStart } from 'kibana/public';
+import { OBSERVABILITY_OWNER, SECURITY_SOLUTION_OWNER } from '../../common/constants';
+
+export type CasesOwners = typeof SECURITY_SOLUTION_OWNER | typeof OBSERVABILITY_OWNER;
+
+export const canUseCases =
```

Comment:
Could you please add a comment above the function to explain the assumptions on the decisions when returning the capabilities? For example that if one of the solutions has `crud_cases` it will return true even if the others solutions do not have `crud` permissions.

Reply:
got it!